src/models/hooks/sessionReportPilot.test.js

```diff
@@ -421,6 +422,80 @@ describe('sessionReportPilot hooks', () => {
       await afterUpdate(mockSequelize, instance, mockOptions);
       expect(trSessionCompleted).not.toHaveBeenCalled();
     });
+
+    it('sets the SessionReportPilot status to complete if both owner and poc are complete', async () => {
```

Comment:
This seems like an instance where we'd want a bit more robust of a test, right? Like maybe one with some special characters or even one that interacts with the actual database?

Reply:
Disregard, as you are reworking this